internal/tools/aws/aim.go

```diff
@@ -0,0 +1,248 @@
+package aws
```

Comment:
Typo in the filename? "aim.go" vs "iam.go"?

Reply:
Yep, thanks!